### ISSUE TITLE
[FEATURE] Allow changing icons of folders

### ISSUE THREAD

**Issue by BigDaddyBubba37** [Deletescape-Media/Lawnchair#4060]
### Describe the feature

Currently unable to change the icon for the folders 

### Why would it be useful to add?

Would help make the home screens cleaner. 

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

![Screenshot_20240208-101017](https://github.com/LawnchairLauncher/lawnchair/assets/154076593/f2618c38-6806-422d-802e-4ea17b1e421b)

### Additional context

This feature is in nova. But I prefer using lawnchair. It feels so much cleaner/leaner

**Comment by StarGW-net** [comment]
Yup, this is my must have feature for a launcher. Makes a huge difference to usability.

Nova looks so much better:

![Screenshot_20240409_132435_Nova7](https://github.com/LawnchairLauncher/lawnchair/assets/19812968/5166170b-8bf0-486c-9705-8fb30af8cb6d)

Most of the Icons are folders containing logically grouped apps.

**Comment by butjer** [comment]
If this feature was possible, I change from Lawnchair 2 to 14.
Also mentioned in #2940.
It's a big wish from serveral users.

**Comment by phillipprado** [comment]
Would absolutely love this feature. It might be the only thing left I'm missing from every other launcher.

**Comment by Happyclem** [comment]
Adding to this, even adjusting the transparency for the group (like getting rid of it but keeping all the app icons in small bunch) or having just an outline would make screens look cleaner.

**Comment by ghost** [comment]
+1 from me. I miss this feature as well.

**Comment by ItsGoldMonke** [comment]
+1, I really need this

**Comment by Matth7878** [comment]
Being able to change how folder icons display their contents could already be a huge improvement. For instance stacking apps icon inside would make it much cleaner than showing them as a grid. For instance : 

![Image](https://github.com/user-attachments/assets/d86a5f8e-7290-47e5-b700-8c2bfaf5b928)

**Comment by kajdo** [comment]
now that nova dies (https://teslacoilapps.com/nova/solong.html) this would be a nice timing i'd say ... please

**Comment by butjer** [comment]
I would change from lawnchair 2 today
if folder icons are implemented.
Long awaiting feature.

**Comment by Bosswaffle** [comment]
+1 I'm testing out alternatives to nova, and will probably stick with whichever one lets me use custom folder icons!

**Comment by StarGW-net** [comment]
I gave up waiting for this simple feature to be re-implemented. Went back to the stock launcher.

Instead I found a nifty utility JINA Drawer that almost does what I want. Groups apps and uses a custom widget to mimic folders. Can use with any launcher.

https://play.google.com/store/apps/details?id=com.mobeedom.android.jinaFS

**Comment by PhilZ-cwm6** [comment]
This was a big disappointment when I tried to move from Nova prime
Add to missing gestures and the recent/updated tabs in apps search, it makes Launchair much less appealing

Keep up the good work and hope you can add these

**Comment by noraj** [comment]
This core feature was existing in Lawnchair v2 (legacy), this days I tried Lawnchair v15 so I get a maintained version. But I was dusapointed that there was no migration tool so I need recreate my whole configuration manually. While doing that I was even more desapointed there is no folder icon feature so all my hole screens look ugly.
